README.md

```diff
@@ -6,7 +6,7 @@
 [tg-badge]: https://img.shields.io/endpoint?color=neon&logo=telegram&label=chat&style=flat-square&url=https%3A%2F%2Ftg.sumanjay.workers.dev%2Fdojoengine
 [tg-url]: https://t.me/dojoengine
 
-Dojo is a developer friendly framework for building **provable** Games, Autonomous Worlds and other Applications that are natively composable, extensible, permissionless and persistent. It is an extension of [Cairo](https://www.cairo-lang.org/), an efficiently provable language, that supports generation of zero-knowledge proofs attesting to a computations validity and enables exponential scaling of onchain computation while maintaining the security properties of Ethereum.
+Dojo is a developer friendly framework for building **provable** Games, Autonomous Worlds and other Applications that are natively composable, extensible, permission-less and persistent. It is an extension of [Cairo](https://www.cairo-lang.org/), an efficiently provable language, that supports generation of zero-knowledge proofs attesting to a computations validity and enables exponential scaling of onchain computation while maintaining the security properties of Ethereum.
```

Comment:
i dont have a phd in the english language but i think 'permissionless' is still considered correct.